boards/arm/mimxrt1050_evk/Kconfig.defconfig

```diff
@@ -53,9 +53,6 @@ endif # NETWORKING
 
 if LVGL
 
-config LVGL_DISPLAY_DEV_NAME
```

Comment:
Who deprecated it and when?

Reply:
[Here](https://github.com/zephyrproject-rtos/zephyr/pull/38306#discussion_r748644311) @mbolivar-nordic  mentioned that it was deprecated.

Reply:
@DineshDK03 That was meant in general, look at the changes in #40765, now option LVGL_DISPLAY_DEV_NAME obtain value from devicetree only and hopefully will be removed after some LVGL adaption code rework. You probably want to rebase on latest main branch.

Reply:
@jfischer-no  Updated the LVGL_DISPLAY_DEV_NAME changes along with some minor changes mentioned in the driver ek79652.c except refresh APIs. expecting your comments on refresh API's and changes based on refresh API.